pom.xml

```diff
@@ -888,6 +888,11 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>io.github.aliyunmq</groupId>
+                <artifactId>rocketmq-grpc-netty-codec-haproxy</artifactId>
+                <version>1.0.0</version>
+            </dependency>
```

Comment:
Would it be better to copy the code instead of introducing a library?

Reply:
The haproxy library contains 10 classes and has a lot of code. In order to be able to follow the upgrade of the grpc-netty version in the future, it may be better to introduce it by shade